tests/e2e/models/Navigation.js

```diff
@@ -29,3 +32,14 @@ class DesktopNav {
 
   }
 }
+
+class MobileNav{
+  constructor(mobileMenu, topToolbar){
+    this.hamburgerButton = topToolbar.find('.BRmobileHamburger');
+    this.menuSearchButton = mobileMenu.find('[data-target="#mm-4"]');
```

Comment:
Yikes! That's an ugly selector 😅 Could you create a new issue to add nicer selectors to these+update these tests?

Reply:
https://github.com/internetarchive/bookreader/issues/323